thrust/thrust/zip_function.h

```diff
@@ -10,8 +10,8 @@
 #include <thrust/detail/config.h>
 #include <thrust/detail/cpp11_required.h>
 #include <thrust/detail/modern_gcc_required.h>
-
-#if THRUST_CPP_DIALECT >= 2011 && !defined(THRUST_LEGACY_GCC)
+// remove?
```

Comment:
While we officially do not support those old gcc compilers anymore there is no benefit in intentionally breaking support right now, so I would leave that as is
```suggestion

```

Reply:
In the sense as make it `#if !defined(THRUST_LEGACY_GCC)`